src/form-control/form-control.js

```diff
@@ -83,17 +93,57 @@ export default class FormControl extends React.Component<
       }
     }
 
+    let labelEndEnhancer = this.props.labelEndEnhancer;
+    if (counter) {
```

Comment:
Should this override developer-provided `props.labelEndEnhancer`? Maybe include a check to apply this if-block only when `props.labelEndEnhancer` is falsey

Reply:
This is a good point